tutorials/jms-output-exactlyonce/pom.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  
+  <groupId>com.example</groupId>
+  <version>1.0-SNAPSHOT</version>
+  <artifactId>malhar-example-jmsactivemq-output</artifactId>
+  <packaging>jar</packaging>
+
+  <!-- change these to the appropriate values -->
+  <name>My Apex Application</name>
+  <description>My Apex Application Description</description>
+
+  <properties>
+    <!-- change this if you desire to use a different version of Apex Core -->
+    <apex.core.version>3.6.0-SNAPSHOT</apex.core.version>
+    <apex.malhar.version>3.8.0-SNAPSHOT</apex.malhar.version>
```

Comment:
As discussed, the actual fix in the JMS output operator in malhar is not in, right?